source/manage/telemetry.rst

```diff
@@ -36,32 +40,44 @@ The following data is collected once every 24 hours:
 - Date and time of the last check for security updates
 - The location of the Amazon Cloudfront server used for telemetry data
 
-To opt out, disable the feature in **System Console > Environment > SMTP**. When the feature is disabled, you will not receive any security alerts.
+Opt out
+~~~~~~~
+
+To opt out, you can disable this security update check feature for self-hosted deployments in the System Console by going to **Environment > SMTP > Enable Security Alerts**. See the `enable security alerts </configure/environment-configuration-settings.html#enable-security-alerts>`__ documentation for details. When this feature is disabled, you will not receive any security alerts.
```

Comment:
We say "self-hosted servers" in the previous note but "self-hosted deployments" here. 

This is a great opportunity for me to standardize this provided we're referring to the same things. Or, if the server is different to the deployment (i.e., we're talking about discrete physical servers within a deployment of Mattermost that each need to be configured vs. the actual overarching Mattermost deployment) that's also OK. 

I'll work on the definitions/adding it to the list once we've decided on the terminology. :)